Certificates/cert-gen.py

```diff
@@ -0,0 +1,103 @@
+# Requires inkscape in path and PDF module
+import argparse
+import csv
+import numpy as np
+import os
+import pathlib
+from PDF import PdfFileReader, PdfFileWriter
+from subprocess import run
```

Comment:
For clarity I'd encourage importing the whole `subprocess` module and then using it as `subprocess.run` -- that makes it clear what `run` does without needing to look at the definition.

Reply:
Further to that, I'd encourage the use of [`subprocess.check_call`](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) so that any errors from Inkscape are thrown as errors in Python.

Reply:
Fair enough :)
Used `subprocess.run(..., check=True)` because apparently this does the same thing.